src/components/PaginationFooter/constants.ts

```diff
@@ -0,0 +1,4 @@
+const PAGINATION_HEIGHT = 40;
```

Comment:
it's better to move this into `src/constants` because since it's loaded one time (for HeadCellMenu) other constants also will be used.

on the flip side, i dont think that a variable will hurt the treeShaking process

Reply:
Will move it